### ISSUE TITLE
Please leave your feedback here

### ISSUE THREAD

**Issue by ezraholm50** [ezraholm50/MediaCenter#1]
@enoch85

**Comment by enoch85** [comment]
NICE! :+1: But please get the Let's Encrypt script from the existing repo as they might change, and it's unnecessary to have it in two places. 

Would also be nice if you could write a post about it once it's done. Great work man!

**Comment by enoch85** [comment]
Also, we should create a new repo for Tech and Me if you ask me. But that's for later.

**Comment by ezraholm50** [comment]
Yeah I will update letsencrypt soon, first polishing it. Need it finished first. We now have a graphical interface within the CLI! @enoch85
